tests/test_curate_coding.py

```
@@ -780,6 +787,29 @@ def test_wrap_record_curation_is_output_idempotent(self):
         self.assertEqual(second_manifest["action"], "unchanged")
         self.assertEqual(second_manifest["step_counts"]["migrated"], 0)
 
+    def test_wrap_record_initial_migration_passes_verification(self):
+        source = wrap_record([visible_step()])
+
+        curated, manifest = curate_episode(source)
```

Ruff (0.16.2)

[warning] Unpacked variable `curated` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)